node/index.js

```diff
@@ -127,12 +127,15 @@ app.post(
 
         client.createPayment(
           recipientId,
-          'payment_ref',
+          'paymentref',
           {
             value: 12.34,
             currency: 'GBP',
           },
           function (error, createPaymentResponse) {
+            if (error != null) {
+              prettyPrintResponse(error);
```

Comment:
return the error after printing the error? otherwise it looks like we just continue on with the logic here

```
    if (error != null) {
      prettyPrintResponse(error); 
      return response.json({ 
        error: error, 
      }); 
    } 
```

Reply:
done